src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -139,6 +162,11 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     public static class CSharpExtensions
     {
+        /// <summary>
+        /// Determines if the given raw kind value belongs to the C# <see cref="SyntaxKind"/> enumeration.
+        /// </summary>
+        /// <param name="rawKind">The raw value to test.</param>
+        /// <returns><c>true</c> when the raw value belongs to the C# syntax kind; <c>false</c> otherwise.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true"/> when the raw value belongs to the C# syntax kind; otherwise, <see langword="false"/>.</returns>
```